src/annotator/adder.js

```diff
@@ -193,6 +189,34 @@ export class Adder {
     return { top, left, arrowDirection };
   }
 
+  /**
+   * Find the zIndex of five nearby elements (left-top, left-bottom, middle-center,
+   * right-top, right-bottom) and use the greatest.
+   *
+   * @param {number} left - Horizontal offset from left edge of viewport.
+   * @param {number} top - Vertical offset from top edge of viewport.
+   * @return {number} - greatest zIndex
+   */
+  _findZindex(top, left) {
+    const width = this._width();
+    const height = this._height();
+
+    // Unique elements and ancestors
+    const elements = new Set([
```

Comment:
Is it OK to use `Set` to get all the unique elements?

Reply:
In general you can use the `Set` and `Map` classes in our code. We have polyfills for these.